### PR TITLE
Add to ignore if scanning too often

### LINKED ISSUE #4497: Looped spin fort and move to fort. Endless "Possibly searching too often, take a rest."
### Expected Behavior

Spin the current fort and go to another (like it was some days before)

### Actual Behavior

Looped move to fort / spinning. Bot can be stuck in this state for like 30 minutes continuously trying to spin the fort.

### Your FULL config.json (remove your username, password, gmapkey and any other private info)

http://pastebin.com/2gc6gaaP

### Output when issue occurred
2016-08-21 22:07:29,914 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Vse Dla Fronta - 0.04km
2016-08-21 22:07:31,357 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:31,363 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:33,051 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:33,060 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Vse Dla Fronta - 0.04km
2016-08-21 22:07:35,968 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:35,974 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:37,807 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:37,813 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:40,849 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:40,857 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Vse Dla Fronta - 0.04km
2016-08-21 22:07:42,715 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:42,722 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:44,585 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:44,591 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:46,203 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:46,209 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Vse Dla Fronta - 0.04km
2016-08-21 22:07:48,157 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:48,163 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:50,885 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:50,891 [MoveToFort] [INFO] [moving_to_fort] Moving towards pokestop Vse Dla Fronta - 0.04km
2016-08-21 22:07:52,325 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:52,331 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:54,024 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:54,030 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:55,895 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.
2016-08-21 22:07:55,904 [MoveToFort] [INFO] [arrived_at_fort] Arrived at fort.
2016-08-21 22:07:57,741 [  SpinFort] [INFO] [pokestop_searching_too_often] Possibly searching too often, take a rest.

### Steps to Reproduce

Install using setup.sh from dev branch and just launch bot with my config. After 1-2 min bot will stuck

### Other Information
OS: 
MacOS
Branch: 
dev
Git Commit: 
3fcc2901c0f1afc14225c5455c6d2bebbedf5500
Python Version: 
Python 2.7.12 
Any other relevant files/configs (eg: path files)

### PR DESCRIPTION
**Pull request by mjmadsen**
- Fixed some whitespace issues - would those have caused the if/else to not parse as we would expect?
- Added stop to ignore list if we get a search too often error.
- Increased default spinfort wait times

Hopefully fixes #4497 and reoccurring similar problems

### CODE REVIEW

File: pokemongo_bot/cell_workers/spin_fort.py [hunk 1]
```diff
@@ -150,27 +150,28 @@ def work(self):
                         'softban',
                         formatted='Probably got softban.'
                     )
-                with self.bot.database as conn:
-                    c = conn.cursor()
-                    c.execute("SELECT COUNT(name) FROM sqlite_master WHERE type='table' AND name='softban_log'")
-                result = c.fetchone()        
-
-                while True:
```
Comment (solderzzc) [inline pokemongo_bot/cell_workers/spin_fort.py]: Why there was a while True: ? It seems just one time run, why placed the `while` here

Comment (mjmadsen) [inline pokemongo_bot/cell_workers/spin_fort.py]: I'm not terribly should about this DB code.

**Comment by solderzzc** [comment]
Yes, your fix will improve the real soft ban log.

**Comment by mjmadsen** [comment]
@solderzzc That look better. Yeah, the original source had a break after each if and appeared to not need looping. Removed.

**Comment by solderzzc** [comment]
@mjmadsen  thanks a lot, got merged.

**Pull request merged**